netbout-web/src/main/java/com/netbout/misc/Ports.java

```diff
@@ -50,16 +50,23 @@
         new ConcurrentSkipListSet<Integer>();
 
     /**
+     * Hide the default no args constructor from the utility class.
```

Comment:
@danff please see http://www.yegor256.com/2014/05/05/oop-alternative-to-utility-classes.html#utility-classes-are-evil and watch https://www.youtube.com/watch?v=psrp3TtaYYI, we cannot leave it as utility class, but I'm OK for a puzzle